frontend/src/components/pages/CampCreation/ScheduleSession/index.tsx

```diff
@@ -0,0 +1,18 @@
+import { Box, Text } from "@chakra-ui/react";
+import React, { useState } from "react";
+import SessionsCalendar from "./SessionsCalendar";
+
+const ScheduleSessionsPage = (): React.ReactElement => {
+  const [scheduledSessions, setScheduledSessions] = useState<Array<Array<string>>>([]);
```

Comment:
Is `scheduledSessions` supposed to be a `Array<Array<string>>` or is this just for dummy data? Wouldn't it be better for this to be a `CampSession[]`?